api/src/main/java/openconsensus/trace/data/AttributeValueType.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2019, OpenConsensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package openconsensus.trace.data;
+
+/**
+ * An enum that represents all the possible value types for an {@code AttributeValue}.
+ *
+ * @since 0.1.0
+ */
+public enum AttributeValueType {
```

Comment:
I would make AttributeValueType an inner class of the AttributeValue.Type, what do you think?